src/azul/service/storage_service.py

```diff
@@ -66,6 +76,27 @@ def get_presigned_url(self, key: str, file_name: Optional[str] = None) -> str:
     def create_bucket(self, bucket_name: str = None):
         self.client.create_bucket(Bucket=(bucket_name or self.bucket_name))
 
+    def put_filename_tag(self, object_key: str, file_name: str):
+        deadline = time.time() + 60
+        tagging = {'TagSet': [{'Key': 'azul_file_name', 'Value': file_name}]}
```

Comment:
The tag name (aka "tag key") should be extracted to a class attribute, like the suffix was before.